packages/now-cli/src/util/dev/builder.ts

```diff
@@ -280,7 +280,6 @@ export async function executeBuild(
       path = extensionless;
     }
 
-    delete output[path];
```

Comment:
This line was not doing anything since it's being re-assigned on the line below.